manipulator:
  kinematics_solver: kuka_kr210_manipulator_kinematics/IKFastKinematicsPlugin
  kinematics_solver_attempts: 10
  kinematics_solver_search_resolution: 0.01
  kinematics_solver_timeout: 0.01
